modules/openapi-generator/src/main/resources/META-INF/services/org.openapitools.codegen.CodegenConfig

```diff
@@ -107,3 +107,5 @@ org.openapitools.codegen.languages.TypeScriptJqueryClientCodegen
 org.openapitools.codegen.languages.TypeScriptNodeClientCodegen
 org.openapitools.codegen.languages.TypeScriptRxjsClientCodegen
 org.openapitools.codegen.languages.FsharpGiraffeServerCodegen
+
+org.openapitools.codegen.languages.KotlinVertxServerCodegen
```

Comment:
@Wooyme Can you put this according to alphabetical order?

Please move `FsharpGiraffeServerCodegen` (line 109) to the proper location in this file based on alphabetical order if you don't mind.